client/modules/PostEditor/PostEditor.tsx

```diff
@@ -119,6 +119,12 @@ function PostEditor(props: types.Props) {
     }
   );
 
+  const firstCategory = categories?.length > 0 ? _.head(categories) : null;
+  const actualCategory =
```

Comment:
Please rename `actualCategory` to `selectedCategory`. Also, don't assign category ID here, rather find actual category object - for example by using `_.find`